### ISSUE TITLE
[Bug]: FluentUI ContextualMenu Accessibility Issue aria-allowed-attr

### ISSUE THREAD

**Issue by mzavala722** [microsoft/fluentui#27593]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
Package version: @fluentui/react: 8.93.1
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://developer.microsoft.com/en-us/fluentui#/controls/web/contextualmenu

## Repro Steps
- Navigate to the documentation for the ContextualMenu component: https://developer.microsoft.com/en-us/fluentui#/controls/web/contextualmenu
- Find the "ContextualMenu with checkable menu items and toggleable split button" example
- Open the contextual menu
- Run the Accessibility Insights for Web fast pass.

## Actual Behavior
Implementations of the FluentUI ContextualMenu are currently failing the Accessibility Insights for Web fast pass. See screenshots and attached logs.

![image](https://user-images.githubusercontent.com/130993509/232533646-f2a74a3e-5922-401d-8e1b-763583d2ffd6.png)
![image](https://user-images.githubusercontent.com/130993509/232533755-fc37a2b9-134a-43a5-ab22-f60c961ced25.png)

## Expected Behavior
The ContextualMenu component should pass the Accessibility Insights for Web fast pass without any accessibility issues.

### Logs

```shell
Accessibility Insights for Web fast pass failure details:

Title: WCAG 4.1.2: Ensures ARIA attributes are allowed for an element's role (.splitContainer-422[aria-posinset="6"][role="menuitemcheckbox"])
Tags: Accessibility, WCAG 4.1.2, aria-allowed-attr

Issue: Ensures ARIA attributes are allowed for an element's role (aria-allowed-attr - https://accessibilityinsights.io/info-examples/web/aria-allowed-attr)

Target application: Fluent UI - Controls - React - ContextualMenu - https://developer.microsoft.com/en-us/fluentui#/controls/web/contextualmenu

Element path: .splitContainer-422[aria-posinset="6"][role="menuitemcheckbox"]

Snippet: <div role="menuitemcheckbox" class="splitContainer-422" aria-disabled="false" aria-expanded="false" aria-haspopup="true" aria-posinset="6" aria-setsize="10" tabindex="-1" data-is-focusable="true">

How to fix: 
Fix any of the following:
  ARIA attribute is not allowed: aria-expanded="false"

Environment: Microsoft Edge version 112.0.1722.39

====

This accessibility issue was found using Accessibility Insights for Web 2.38.1 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
```

### Requested priority

Blocking

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by khmakoto** [comment]
Hi @mzavala722, this seems like it is a false positive in the `axe-core` checks.

You can see that `aria-expanded` is an allowed state for the `menuitemcheckbox` role in the [ARIA spec](https://w3c.github.io/aria/#menuitemcheckbox).

We have [filed an issue](https://github.com/dequelabs/axe-core/issues/3992) on `axe-core` regarding this, so I'll be resolving this issue as `Resolution: External`.

Let us know if there is anything else we can do to help!
